.github/workflows/upload-sdk-artifact.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 env:
   NODE_VERSION: 20.x
-  SDK_VERSION: latest
+  SDK_VERSION: 0.0.1
```

Comment:
Is this correct?

Reply:
It's a fix. Emil reported that pip requires a numeric version to install the SDK package

Reply:
Ok. Let's keep as it is and think a more meaningful value later